README.md

```diff
@@ -1,4 +1,20 @@
-# advocacy.mozilla.org
+# Overview
+
+This is the source code for [advocacy.mozilla.org]()
+
+Currently there are three parts to the site.
+
+[Homepage](advocacy.mozilla.org) and [Open Web Fellows](https://advocacy.mozilla.org/open-web-fellows/) are an index.html file statically generated by a server at build time. This single index page then renders the requested page on the client side.
+
+[Encrypt Campaign](https://advocacy.mozilla.org/encrypt/) doesn't use an html file and nothing is built on the server, instead the server generates the pages as requested and passed to the client. The result page is cached on the server in memory after the first request, so the next request is not generated again.
+
+# Getting Started
+
+## Dependencies
+
+To get a local version of the site running, you'll need to have [git](http://git-scm.com/) and [node](http://nodejs.org/) installed on your local machine.
```

Comment:
Note that this is technically not true, as you don't need `git`, you could just download the repo as archive, unpack it, and then run that. It's only if you want to *contribute* that `git` becomes a necessity.